packages/sdk/src/sdk/errors.ts

```diff
@@ -140,3 +140,29 @@ export class ThreadAlreadyExistsError extends MessagingError {
     );
   }
 }
+
+export abstract class IdentityError extends DialectSdkError {}
+
+export class SNSIdentityError extends IdentityError {
```

Comment:
I think we violate dependency inversion principle here: sdk package should not contain sns/cardinal/civic deps, wdyt?

Reply:
Actually, you are right. They are specific errors. I separated them from sdk to their own packages 👍